.github/workflows/ci-build-macos.yaml

```diff
@@ -39,6 +39,13 @@ jobs:
     timeout-minutes: 120
 
     steps:
+      - name: cancle previours runs
+        uses: apache/airflow-cancel-workflow-runs@953e057dc81d3458935a18d1184c386b0f6b5738
+        with:
+          cancelMode: allDuplicates
```

Comment:
Based on the [documentation of apache/airflow-cancel-workflow-runs](https://github.com/apache/airflow-cancel-workflow-runs#inputs-and-outputs), I'd assume that this should be `duplicates` instead of `allDuplicates`. 
Since `duplicates` is the default value, the `cancelMode` parameter can be omitted.

Reply:
some clarification: `allDuplicates` means "Cancels duplicate runs from all running workflows.". I don't think this makes sense for per-workflow cancellation. the default cancelMode `duplicates` means "Cancels duplicate runs from the same repo/branch as current run."